src/NuGet.Core/NuGet.ProjectModel/RestoreAuditProperties.cs

```diff
@@ -22,12 +21,19 @@ public class RestoreAuditProperties : IEquatable<RestoreAuditProperties>
         /// <value>low, moderate, high, critical</value>
         public string? AuditLevel { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating which audit mode to use.
+        /// </summary>
+        /// <value>direct, transitive</value>
```

Comment:
Either delete this line or:

```suggestion
        /// <value>direct, all</value>
```